packages/components/psammead-figure/src/index.stories.jsx

```diff
@@ -27,20 +30,23 @@ storiesOf('Figure', module)
   )
   .add(
     'containing Image, ImagePlaceholder, Copyright and Caption',
-    () => (
```

Comment:
It would be good to set the language value once for the three items `caption` `captionOffscreen` and `copyrightOffscreen`, because currently to check the Caption works in `Swahili`, I need to manually go into each tab, check the checkbox and then select `Swahili`. 

<img width="589" alt="Screen Shot storybook" src="https://user-images.githubusercontent.com/3028997/55945116-10be1380-5c42-11e9-860d-bd6913213a08.png">

Reply:
Preset language is now only used for the Caption due to problems with default text and how the inputProvider function is written. Will raise an issue to address the inputProvider and also one for this in the future.